spec/factory_specs/stripe/stripe_cards_spec.rb

```diff
@@ -17,10 +17,4 @@
 
     expect(StripeMockHelper.instance.customers.count).to eq 1
   end
-
-  it 'has created a single source on the customer' do
-    create(:stripe_card_base)
-
-    expect(StripeMockHelper.instance.customers.first[1][:sources][:total_count]).to eq 1
-  end
```

Comment:
This was using an internal data structure on StripeMock. It was fine for some early testing but isn't compatible with new versions of StripeMock so we should just remove it.